tests/resources/dataplex_gcs_metadata_expected_configs.json

```diff
@@ -27,6 +27,7 @@
     "dataset_name": "<your_dataplex_bigquery_dataset_id>",
     "project_name": "<your-gcp-project-id>",
     "resource_type": "CLOUD_STORAGE",
+    "partition_fields": null,
```

Comment:
can we change this GCS asset to one that has a partition field? I'd like to have a test for this.

Reply:
I am adding a separate test for the partitioned gcs asset, this is for the existing one without partitioning.